app/Libraries/Search/BeatmapsetSearchRequestParams.php

```diff
@@ -147,6 +150,10 @@ private function getDefaultSort(string $order): array
             return [new Sort('last_update', $order)];
         }
 
+        if (in_array('can_bundle', $this->extra, true)) {
+            return [new Sort('bundled', $order)];
+        }
```

Comment:
Shouldn't a filter option filter the results ?_?

Reply:
filter by `can_bundle` and default sort by `bundled` so the currently bundled maps are always at the top. i thought it'd be nice so you can see what new players are downloading right now, on second thought I'm not sure how useful that is since there's no visual indicator for maps that are bundled

Reply:
This seems confusing given there's no UI to change the ordering, on top of it overriding the default sort or whatever category filter is being used. If `can_bundle` and `bundled` are going to be flags, it would make more sense for them to have separate filter options instead of being a sort order.